### PR TITLE
allow open in new tab

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?222501
@orangejenny

### CODE REVIEW

File: corehq/apps/app_manager/templates/app_manager/app_view.html [hunk 1]
```diff
@@ -89,7 +89,11 @@
                 $loading = $main.find(".hq-loading").remove(),
                 $loadingError = $main.find(".hq-loading-error").remove();
 
-            $('#demand-releases').on('show.bs.tab', function () {
+            $('#demand-releases').on('show.bs.tab', _.throttle(function () {
```
Comment (orangejenny) [inline corehq/apps/app_manager/templates/app_manager/app_view.html]: what's the purpose of the throttle? to allow double-clicking?

**Comment by orangejenny** [comment]
Seems fine, merging.

Did you have a chance to test the other "links" in app manager that don't go to new pages? I'd guess that the multimedia/languages/settings/copy/delete links would all behave the same as the deploy link. Feel free to kick this over to me if you don't want to look into this further.

File: corehq/apps/app_manager/templates/app_manager/app_view.html (hunk 1 above)
Comment (snopoke) [inline corehq/apps/app_manager/templates/app_manager/app_view.html]: When I was testing this every click execute the listener twice. I verified that the listener is only added once and that the events for each execution were identical. Couldn't figure out why the 2nd execution was happening.

**Comment by snopoke** [comment]
I didn't check the other links. I think it's less likely someone would want to open them in a new tab but I guess it's good to be consistent. If you could take a look that would be great. Thanks.

**Comment by orangejenny** [comment]
Cool, will add to backlog.

**Pull request merged**